packages/zoe/src/zoeService/startInstance.js

```diff
@@ -0,0 +1,225 @@
+// @ts-check
+
+import { assert, details as X } from '@agoric/assert';
+import { E } from '@agoric/eventual-send';
+import { makePromiseKit } from '@agoric/promise-kit';
+import { makeWeakStore as makeNonVOWeakStore } from '@agoric/store';
+import { Far } from '@agoric/marshal';
+
+import { makeZoeSeatAdminKit } from './zoeSeat';
+import { makeHandle } from '../makeHandle';
+import { handlePKitWarning } from '../handleWarning';
+
+/**
+ * @param {Promise<ZoeService>} zoeServicePromise
+ * @param {MakeZoeInstanceStorageManager} makeZoeInstanceStorageManager
+ * @param {UnwrapInstallation} unwrapInstallation
+ * @returns {StartInstance}
+ */
+export const makeStartInstance = (
+  zoeServicePromise,
+  makeZoeInstanceStorageManager,
+  unwrapInstallation,
+) => {
+  /** @type {StartInstance} */
+  const startInstance = async (
+    installationP,
+    uncleanIssuerKeywordRecord = harden({}),
+    customTerms = harden({}),
+  ) => {
+    /** @type {WeakStore<SeatHandle, ZoeSeatAdmin>} */
+    const seatHandleToZoeSeatAdmin = makeNonVOWeakStore('seatHandle');
+
```

Comment:
This is a meaningful change. This used to be in zoe.js, but it's better for the weakStore to be instance-specific.

Reply:
Agreed. Resolving.

(Communicating in self-comments like this is good.)